### ISSUE TITLE
Visual Studio cannot properly debug a crashing AppSDK/Reunion app with postmortem debugger

### ISSUE THREAD

**Issue by ivberg** [microsoft/WindowsAppSDK#973]
**Describe the bug**
Visual Studio still cannot properly debug a crashing on startup AppSDK/Reunion app with a postmortem debugger. May or may not be related to running the app and VS as Admin.

Using latest AppSDK 0.8 and VS 16.10.2

**Steps to reproduce the bug**
1. Create a AppSDK 0.8 C# app that has some crash/exception on startup (mine does now after upgrade from 0.5.7 to 0.8)
2. Make that app admin
3. Normally you can attach a debugger but since the crash is on startup you can't. 
3b. You are also blocked by another issue not allowing admin AppSDK apps to be launched/debugged from VS. Could be same root cause? https://github.com/microsoft/WindowsAppSDK/issues/896
4. The app will crash, and VS JIT Debugger UI will launch
5. Select existing VS Admin instance of your app
6. Process will appear to attach and load symbols for a while. However no exception will be shown and the app will appear to be "running" from the debugger. Meanwhile the app UI will be crashed/frozen as if the process is being debugged.
7. No matter how long you wait, VS debugger does not show exception

**Expected behavior**
You can debug startup AppSDK app crashes and debug them with the post-mortem JIT debugger

**Screenshots**

**Version Info**

    PackageReference Include="Microsoft.ProjectReunion" Version="0.8.0" />
    PackageReference Include="Microsoft.ProjectReunion.Foundation" Version="0.8.0" />
    PackageReference Include="Microsoft.ProjectReunion.WinUI" Version="0.8.0" />

VS 16.10.2

NuGet package version: 
See above

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (22000.41)              | Yes  |

**Additional context**

**Comment by asklar** [comment]
@auott, does this ring a bell?

**Comment by ivberg** [comment]
Appears to be an issue that only repros on latest Win11 Insider Builds. Bug at OS level. We are investigating

**Comment by riverar** [comment]
WinUI is known busted on Windows 11. https://github.com/microsoft/microsoft-ui-xaml/issues/5313 

You'll have to wait for the next servicing cycle.

**Comment by ivberg** [comment]
yes @riverar unfortunately WinUI 0.8 is broken on Win11 (see my other issues). However, I am pretty sure you are talking about 0.8 as Win11 is not broken on 0.5.7. This issue also repro's on 0.5.7 and I have serious doubt it will be fixed in 0.8.1 or whatever.

**Comment by andrewleader** [comment]
The crashing on startup for Windows 11 has been fixed with 0.8.1 which is available now. Sounds like this bug is specifically about wanting to be able to debug pre-startup exceptions though, so leaving open for Ben.

**Comment by ivberg** [comment]
I have been following up with @auott on this one. He thinks it's an OS bug and we have filed a bug there

**Comment by andrewleader** [comment]
Awesome thanks, I'll close this then since you have an OS bug tracking it!

**Comment by asklar** [comment]
Tracking: https://task.ms/34464421
